lnrpc/rpc.proto

```diff
@@ -2797,3 +2810,91 @@ message BakeMacaroonResponse {
     /// The hex encoded macaroon, serialized in binary format.
     string macaroon = 1 [json_name = "macaroon"];
 }
+
+message ChannelReportsRequest{
+    /*
+    Start time is beginning of the range over which the report will be
+    generated, expressed as unix epoch offset in seconds.
+    */
+    uint64 start_time = 1 [json_name = "start_time"];
+
+    /*
+    End time is end of the range over which the report will be
+    generated, expressed as unix epoch offset in seconds.
+     */
+    uint64 end_time = 2 [json_name = "end_time"];
+}
+
+message ChannelReportsResponse{
+    /*
+    Reports is a set of channel reports for all the channels that forwarded
+    htlcs the relevant period.
+    */
+    repeated ChannelReport reports = 1 [json_name = "channel_report"];
+}
+
+message ChannelReport{
+    /*
+    Output is the funding outpoint of the channel this report has been generated
+    for.
+    */
+    OutPoint outpoint = 1 [json_name = "channel_id"];
```

Comment:
json name `chan(nel)_point` perhaps

Reply:
Rename both to `channel_outpoint`? Seems weird to have them be different.